apps_ci/version_bump.py

```diff
@@ -43,6 +42,6 @@ def rename_versioned_dir(version: str, new_version: str, train_name: str, app_di
 
     if new_versioned_dir.is_dir():
         verrors.add('version_bump', f'App {app_dir.name!r} library with version {new_version!r} already exists')
-    curr_versioned_dir.rename(new_versioned_dir)
+    verrors.check()
 
-    print(f'[\033[92mOK\x1B[0m]\tUpdated app {app_dir.name!r} library from {version!r} to {new_version!r}')
```

Comment:
removed print, as there is no reason to do so here, we should only do that on the main script